gtfs-validator/main/src/test/java/org/mobilitydata/gtfsvalidator/validator/StopTimeIncreasingDistanceValidatorTest.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2020 Google LLC, MobilityData IO
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.validator;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mobilitydata.gtfsvalidator.notice.DecreasingStopTimeDistanceNotice;
+import org.mobilitydata.gtfsvalidator.notice.NoticeContainer;
+import org.mobilitydata.gtfsvalidator.table.GtfsStopTime;
+import org.mobilitydata.gtfsvalidator.table.GtfsStopTimeTableContainer;
+
+@RunWith(JUnit4.class)
+public class StopTimeIncreasingDistanceValidatorTest {
+  private final GtfsStopTime stopTime1 = new GtfsStopTime.Builder()
+                                             .setCsvRowNumber(1)
+                                             .setTripId("trip1")
+                                             .setStopId("1001")
+                                             .setStopSequence(1)
+                                             .setShapeDistTraveled(1.0)
+                                             .build();
+  private final GtfsStopTime stopTime2 = new GtfsStopTime.Builder()
+                                             .setCsvRowNumber(2)
+                                             .setTripId("trip1")
+                                             .setStopId("1002")
+                                             .setStopSequence(2)
+                                             .setShapeDistTraveled(1.5)
+                                             .build();
+  private final GtfsStopTime stopTime3 = new GtfsStopTime.Builder()
```

Comment:
perhaps the test will be more readable if the variable would be named according to the intended usage.
For example I would have named this as stop_3_at_1_5

Reply:
Good point, have updated now